### ISSUE TITLE
2 - Convolutional Neural Networks

### ISSUE THREAD

**Issue by datalass1** [datalass1/fastai#7]
image classification, covering several core deep learning concepts that are necessary to get good performance: what a learning rate is and how to choose a good one, how to vary your learning rate over time, how to improve your model with data augmentation (including test-time augmentation). We also share practical tips (such as training on smaller images), an 8-step process to train a world-class image classifier

**Comment by datalass1** [comment]
**Learning Rate**
Basic concept of LR = how quickly we will zoom onto the solution; how quickly or how slowly you want to update the weights or parameters. Paper: https://arxiv.org/pdf/1506.01186.pdf

If LR too low = the model will take too long. LR too high = overshoot and oscillate. 
Solution = learning rate finder.
